### ISSUE TITLE
The quest in ''Kodo hide bag'' in Thunder Bluff  is given as a f...

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#8144]
> The quest in ''Kodo hide bag'' in Thunder Bluff  is given as a followup to the quest ''gathering leather''. In classic it requires you to have the leatherworking profession. On here it doesnt. Dont know if thats a bug or a change

.go xyz -1156.940000 40.703300 145.123000 1 2.847230

**Comment by Bennylavaa** [comment]
This is correct the quest [Gathering Leather](https://www.wowhead.com/classic/quest=768) requires Skinning and [Kodo Hide Bag](https://www.wowhead.com/classic/quest=769) requires Leatherworking.

Also, [Kodo Hide Bag](https://www.wowhead.com/classic/quest=769) is a stand alone quest and does not require [Gathering Leather](https://www.wowhead.com/classic/quest=768) to obtain.


select * from quest_template where entry=769;
                 entry = 769
                 patch = 0
                Method = 2
            ZoneOrSort = 1638
              MinLevel = 4
              MaxLevel = 0
            QuestLevel = 10
                  Type = 0
       RequiredClasses = 0
         RequiredRaces = 0
         RequiredSkill = 165
    RequiredSkillValue = 10
     RequiredCondition = 0
   RepObjectiveFaction = 0
     RepObjectiveValue = 0
 RequiredMinRepFaction = 0
   RequiredMinRepValue = 0
 RequiredMaxRepFaction = 0
   RequiredMaxRepValue = 0
      SuggestedPlayers = 0
             LimitTime = 0
            QuestFlags = 8
          SpecialFlags = 0
           PrevQuestId = 0
           NextQuestId = 0
        ExclusiveGroup = 0
  BreadcrumbForQuestId = 0
      NextQuestInChain = 0
             SrcItemId = 0
          SrcItemCount = 0
              SrcSpell = 0
                 Title = Kodo Hide Bag
               Details = The kodo beasts of Mulgore are strong and sturdy -- traits much respected among us.  If you are skilled in leatherworking and wish the knowledge to make a Kodo Hide Bag, then bring me supplies for my trade.$B$BDo this, and I will share my knowledge with you.
            Objectives = Bring 4 Light Leather and 4 Coarse Thread to Veren Tallstrider in Thunder Bluff.
       OfferRewardText = Ah, good.  Thank you, $N.  My heart swells with pride to see youths interested in the craft of leatherworking.
      RequestItemsText = Do you have my supplies, $N?
               EndText = 
        ObjectiveText1 = 
        ObjectiveText2 = 
        ObjectiveText3 = 
        ObjectiveText4 = 
            ReqItemId1 = 2318
            ReqItemId2 = 2320
            ReqItemId3 = 0
            ReqItemId4 = 0
         ReqItemCount1 = 4
         ReqItemCount2 = 4
         ReqItemCount3 = 0
         ReqItemCount4 = 0
          ReqSourceId1 = 0
          ReqSourceId2 = 0
          ReqSourceId3 = 0
          ReqSourceId4 = 0
       ReqSourceCount1 = 0
       ReqSourceCount2 = 0
       ReqSourceCount3 = 0
       ReqSourceCount4 = 0
    ReqCreatureOrGOId1 = 0
    ReqCreatureOrGOId2 = 0
    ReqCreatureOrGOId3 = 0
    ReqCreatureOrGOId4 = 0
 ReqCreatureOrGOCount1 = 0
 ReqCreatureOrGOCount2 = 0
 ReqCreatureOrGOCount3 = 0
 ReqCreatureOrGOCount4 = 0
         ReqSpellCast1 = 0
         ReqSpellCast2 = 0
         ReqSpellCast3 = 0
         ReqSpellCast4 = 0
      RewChoiceItemId1 = 0
      RewChoiceItemId2 = 0
      RewChoiceItemId3 = 0
      RewChoiceItemId4 = 0
      RewChoiceItemId5 = 0
      RewChoiceItemId6 = 0
   RewChoiceItemCount1 = 0
   RewChoiceItemCount2 = 0
   RewChoiceItemCount3 = 0
   RewChoiceItemCount4 = 0
   RewChoiceItemCount5 = 0
   RewChoiceItemCount6 = 0
            RewItemId1 = 5083
            RewItemId2 = 0
            RewItemId3 = 0
            RewItemId4 = 0
         RewItemCount1 = 1
         RewItemCount2 = 0
         RewItemCount3 = 0
         RewItemCount4 = 0
        RewRepFaction1 = 81
        RewRepFaction2 = 0
        RewRepFaction3 = 0
        RewRepFaction4 = 0
        RewRepFaction5 = 0
          RewRepValue1 = 50
          RewRepValue2 = 0
          RewRepValue3 = 0
          RewRepValue4 = 0
          RewRepValue5 = 0
                 RewXP = 420
         RewOrReqMoney = 0
      RewMoneyMaxLevel = 240
              RewSpell = 0
          RewSpellCast = 0
     RewMailTemplateId = 0
      RewMailDelaySecs = 0
          RewMailMoney = 0
            PointMapId = 0
                PointX = 0
                PointY = 0
              PointOpt = 0
         DetailsEmote1 = 0
         DetailsEmote2 = 0
         DetailsEmote3 = 0
         DetailsEmote4 = 0
    DetailsEmoteDelay1 = 0
    DetailsEmoteDelay2 = 0
    DetailsEmoteDelay3 = 0
    DetailsEmoteDelay4 = 0
       IncompleteEmote = 0
         CompleteEmote = 0
     OfferRewardEmote1 = 1
     OfferRewardEmote2 = 0
     OfferRewardEmote3 = 0
     OfferRewardEmote4 = 0
OfferRewardEmoteDelay1 = 0
OfferRewardEmoteDelay2 = 0
OfferRewardEmoteDelay3 = 0
OfferRewardEmoteDelay4 = 0
           StartScript = 0
        CompleteScript = 0
